ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -476,6 +488,32 @@ export default function ViewQuote() {
             onQuoteDetailsIsOpened={quoteDetailsOpened}
           />
         )}
+        {shouldShowPriceDifferenceWarning && (
+          <div
+            className={classnames(
+              'view-quote__price-difference-warning-wrapper',
+              {
+                high: usedQuote?.priceSlippage?.bucket === 'high',
+                medium: usedQuote?.priceSlippage?.bucket === 'medium',
+              },
+            )}
+          >
+            <ActionableMessage
+              message={
+                <div className="view-quote__price-difference-warning-contents">
+                  {t('swapPriceDifference', priceDifferencePercentage)}
+                  <Tooltip
+                    position="bottom"
+                    theme="white"
+                    html={<>{t('swapPriceDifferenceTooltip')}</>}
```

Comment:
Nit: we can remove the need for the react fragment by using the `title` property of `Tooltip`:
`title={t('swapPriceDifferenceTooltip')}`